docs/source/60_building_apps.md

```diff
@@ -256,6 +256,39 @@ $ spark-submit \
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
 
+### Using ADAM’s ShuffleRegionJoin API
```

Comment:
I'm sorry I don't understand this comment. I looked up Markdown documentation, but this doesn't look like any Markdown syntax I could find.